deployment/refinery-modules/refinery/manifests/init.pp

```diff
@@ -229,6 +226,7 @@
     Class["ui"],
     Class["solr"],
     Class["neo4j"],
+    Class["docker"],
```

Comment:
I don't think supervisor explicitly needs docker, but i don't think we want to start up everything since our app will depend on docker working.